documentation/modules/max-concurrent-vms.adoc

```diff
@@ -0,0 +1,16 @@
+
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="max-concurrent-vms_{context}"]
+= Configuring the controller_max_vm_inflight label
+
+For {rhv-full} migrations, {osp} migrations, VMware warm migrations, or cold VMware migrations to a remote {ocp} cluster the setting `controller_max_vm_inflight` (shown in the UI as *Max concurrent virtual machine migrations*) specifies the maximum number of _disks_ that {project-first} can transfer simultaneously. In these migrations, {project-short} migrates the disks in parallel. This means that if the combined number of disks that you want to migrate is greater than the value of the setting, additional disks must wait until the queue is free, without regard for whether a VM has finished migrating.
```

Comment:
I did more reading and found out that the rhv and osp also do it per VM.
I think the code is not correct and should be changed but this is the current status so I wanted to correct myself.
So the per disk applies only to remote vmware or warm vmware migrations.